### PR TITLE
Add apLCMS

### PR DESCRIPTION
**Pull request by maximskorik**
This PR adds a fork of apLCMS developed by [RECETOX](https://github.com/RECETOX). apLCMS is a tool designated for processing high-resolution LC/MS data.

----
<details>
<summary>Please read the <a href="https://bioconda.github.io/contributor/guidelines.html">guidelines for Bioconda recipes</a> before opening a pull request (PR).</summary>

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>
</details>

### CODE REVIEW

**Comment by maximskorik** [comment]
@BiocondaBot please add label

**Comment by martenson** [comment]
Hello @bioconda/core - would you please review this PR? Thank you. 🙇

File: recipes/r-recetox-aplcms/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+about:
+  home: "{{ github }}"
+  license: GPL-2.0
+  license_file: LICENSE
+  summary: apLCMS generates a feature table from a batch of LC/MS spectra.
+
+extra:
+  maintainers:
+    - RECETOX/conda
```
Comment (bgruening) [inline recipes/r-recetox-aplcms/meta.yaml:67]: that is supposed to be a github username

File: recipes/r-recetox-aplcms/build.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$R CMD INSTALL --build .
```
Comment (bgruening) [inline recipes/r-recetox-aplcms/build.sh:3]: you can put this also in the meta.yaml file as well?

File: recipes/r-recetox-aplcms/meta.yaml (hunk 1 above)
Comment (martenson) [reply recipes/r-recetox-aplcms/meta.yaml:67]: In conda-forge this seems to be accepted (even recommended) as a org team name (https://github.com/conda-forge/staged-recipes/pull/14945#discussion_r639863368) which is also ping-able and can be /imho/ better maintained.

Comment (bgruening) [reply recipes/r-recetox-aplcms/meta.yaml:67]: is that a github group? I mean capital letters? it just looks wired to me, but if its a real group that is fine.

Comment (martenson) [reply recipes/r-recetox-aplcms/meta.yaml:67]: http://github.com/BIOconDA

File: recipes/r-recetox-aplcms/build.sh (hunk 2 above)
Comment (maximskorik) [reply recipes/r-recetox-aplcms/build.sh]: Sure, I've just moved it to the meta.yaml

**Comment by maximskorik** [comment]
@bgruening, thank you for reviewing. It's now ready to get merged.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@bioconda-bot please merge

**Comment by bgruening** [comment]
Awesome, thanks a lot!

**Pull request merged**